src/services/accounts/AccountsValidateService.ts

```diff
@@ -26,16 +34,29 @@ export class AccountsValidateService extends AbstractService {
 
 		if (defaults.allowedEncodedLengths.includes(u8Address.length)) {
 			const [isValid, , , ss58Prefix] = checkAddressChecksum(u8Address);
-
+			network = null;
+			for (const networkParams of allNetworks) {
+				if (networkParams['prefix'] === ss58Prefix) {
+					network = networkParams['network'];
+					break;
+				}
+			}
+			accountId = isHex(address)
```

Comment:
No need for the `let accoundId` either anymore, we can save this directly to a const